lib/go-atscfg/parentdotconfig.go

```diff
@@ -383,258 +390,112 @@ func makeParentDotConfigData(
 			continue
 		}
 
-		isMSO := ds.MultiSiteOrigin != nil && *ds.MultiSiteOrigin
+		// manufacture a topology for this DS.
```

Comment:
Mind putting this in a func? It's a pretty big block, and does a very singular thing. IMO it'd make the code a lot easier to read and work with.

Reply:
moved to CreateTopologies function although there are quite a few args and return variables :(